x/group/migrations/v2/migrate.go

```diff
@@ -36,7 +36,10 @@ func Migrate(
 		accountKeeper.RemoveAccount(ctx, oldAcc)
 
 		// create the group policy account
-		baseAccount, err := group.NewGroupPolicyAccount(oldAcc.GetAddress())
+
+		// TODO
```

Comment:
?

Reply:
Implementing https://github.com/cosmos/cosmos-sdk/pull/13742#issuecomment-1311138512.
So, I reverted the previous implementation in https://github.com/cosmos/cosmos-sdk/pull/13742/commits/d403ab513c8dd510d864b8584a4f0f6ca35b4557. I still need to update the migrations then :(